src/librustdoc/clean/inline.rs

```diff
@@ -199,7 +199,7 @@ crate fn build_external_trait(cx: &DocContext<'_>, did: DefId) -> clean::Trait {
     let generics = (cx.tcx.generics_of(did), predicates).clean(cx);
     let generics = filter_non_trait_generics(did, generics);
     let (generics, supertrait_bounds) = separate_supertrait_bounds(generics);
-    let is_spotlight = load_attrs(cx, did).clean(cx).has_doc_flag(sym::spotlight);
+    let is_spotlight = load_attrs(cx, did).clean(cx).has_doc_flag(sym::notable_trait);
```

Comment:
I would give a warning here about `doc(spotlight)`. In particular I think it should check for either `sym::spotlight` or `sym::notable_trait`, and give a warning for `sym::spotlight`. You might want to make a helper function for that since this is called a few times.

Reply:
How do I make and emit warnings here?